docs/credits.html

```diff
@@ -38,12 +38,29 @@ <h1 style="font-size:50px">The Cortex Command Community Project </h1>
 				Audio Lead - @pawnis <br>
 				Faction Planning Lead - @skaianDestiny <br>
 				Writing Lead - @cave <br>
+				Contributer - @Al Arz <br>
+				Contributer - @CrazyMLC <br>
+				Contributer - @Lugi <br>
+				Contributer - @Tom <br>
+				Contributer - @Just Alex;/ <br>
+				Contributer - @MyNameIsTrez <br>
+				Contributer - @J.H.  <br>
+				Contributer - @HardSphere <br>
+				Contributer - @Wave <br>
+				Contributer - @Borkorus <br>
+				Contributer - @Tatu <br>
+				Contributer - @HeliumAnt <br>
+				Contributer - @Zeta <br>
+				Contributer - @Fil <br>
+				Contributer - @GrandFrag Returns <br>
+				Contributer - @biiaomongegdr <br>
 				<br>
 				Special People<br>
 				Hero of the Revolution - @Weegee ( n/a )<br>
 				Special Spriters - @High Priestess, @Major<br>
-				Website lead - @.Pockets <br>
-				website style - @???? <br>
+				Website - @Tom <br>
+				old website style - @joshy <br>
+				old website - @. ᛈ ᛟ ᚲ ᚲ ᛖ ᛏ ᛋ <br>
```

Comment:
I don't think the "old" signifier is necessary. I would go with a consistent "Website" credit across the board. Or even move them all over to "Contributor"

Reply:
I'm thinking the best solution would be to have contributor vs lead in separate sections, so it looks more like
```
Project Leaders
Thing1 - Person1
Thing2 - Person2
Thing3 - Person3
etc.

Project Contributors
C++ Coding - Person1
INI Coding - Person2
Website Design - Person3
etc.
```

Reply:
yeah. there's a bit of that already. but it's certainly not done yet. I'm also not sure I've gotten everyone

Reply:
Can you spend some time changing it and update this then?